inbox/models/constants.py

```diff
@@ -1,5 +1,3 @@
 # Size constants, set in this file to avoid annoying circular import
 # errors
 MAX_INDEXABLE_LENGTH = 191
-MAX_FOLDER_NAME_LENGTH = MAX_INDEXABLE_LENGTH
-MAX_LABEL_NAME_LENGTH = MAX_INDEXABLE_LENGTH
```

Comment:
Search for `MAX_FOLDER_`, those were reffered in comment with a wrong name in inbox/api/validation.py. Let's remove the stale comment as well.